src/main/java/net/einsteinsci/betterbeginnings/blocks/BlockEnderSmelter.java

```diff
@@ -85,81 +74,6 @@ public static void updateBlockState(boolean lit, World world, BlockPos pos)
 		}
 	}
 
-	public int getMetaFromState(IBlockState state)
-	{
-		return ((EnumFacing)state.getValue(FACING)).getIndex();
-	}
-
-	@Override
-	@SideOnly(Side.CLIENT)
-	public void randomDisplayTick(World world, BlockPos pos, IBlockState state, Random random)
```

Comment:
This function needs to be overridden from `BlockSpecializedFurnace`, since the ender smelter has different particles than the other furnaces.